core/admin.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib import admin
-from core.models import Sector, Theme, Tag, Operation, OperationStep, Review, Source, SourceColumnMap, UpdateHistory, AuditLogEntry
+from core.models import Sector, Theme, Tag, Operation, OperationStep, Review, Source, SourceColumnMap, UpdateHistory, AuditLogEntry, ScheduledEvent, ScheduledEventRunInstance
```

Comment:
This looks very long ... implement a multi-line alternative.

Reply:
Updated